src/StackExchange.Redis/Enums/ServerType.cs

```diff
@@ -30,6 +34,7 @@ internal static class ServerTypeExtensions
         /// </summary>
         public static bool HasSinglePrimary(this ServerType type) => type switch
         {
+            ServerType.Envoyproxy => true,
```

Comment:
Is this one `true` or `false`? I got the impression this was a round-robin desired and not sure about envoy and wasn't sure what _should_ happen. Thoughts?

Reply:
Thanks for catching this, this is resolved